src/main.tsx

```diff
@@ -0,0 +1,11 @@
+import ReactDOM from 'react-dom/client'
+import './index.css'
+import { RouterProvider } from 'react-router-dom'
+import router from './core/router/router.tsx'
+import React from 'react'
```

Comment:
Try to maintain a consistent import approach: start with built-in modules, followed by components, hooks, and CSS styles